### ISSUE TITLE
Statements Menu Item Does Not Load Current History from pg_stat_statements

### ISSUE THREAD

**Issue by bsislow** [dalibo/temboard-agent#565]
We had some database troubles last week and after a database bounce (or around that time), the Statements menu item doesn't show any data after that bounce... this is for the last 7 days:

![image](https://user-images.githubusercontent.com/30243584/155213444-8b28a88f-9d35-46d0-8c3f-cd59f8360391.png)

This is for the last 24 hours:

![image](https://user-images.githubusercontent.com/30243584/155215494-e8268bee-e477-46cd-8097-afd78208cd76.png)

Restarting temboard agent does not help.

The logs only show this error when heading to the Statements screen:

```
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: {'error': 'Invalid session.'}
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: Traceback (most recent call last):
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:   File "/usr/lib/python3.6/site-packages/temboardagent/api.py", line 36, in check_sessionid
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:     session = sessions.get_by_sessionid(xsession)
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:   File "/usr/lib/python3.6/site-packages/temboardagent/sharedmemory.py", line 48, in get_by_sessionid
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:     raise SharedItem_not_found()
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: temboardagent.errors.SharedItem_not_found
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: During handling of the above exception, another exception occurred:
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: Traceback (most recent call last):
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:   File "/usr/lib/python3.6/site-packages/temboardagent/httpd.py", line 131, in response
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:     (code, message) = self.route_request()
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:   File "/usr/lib/python3.6/site-packages/temboardagent/httpd.py", line 253, in route_request
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:     username = check_sessionid(self.headers, self.sessions)
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:   File "/usr/lib/python3.6/site-packages/temboardagent/api.py", line 42, in check_sessionid
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR:     raise HTTPError(401, "Invalid session.")
2022-02-22 14:37:30,681 temboardagent[14567]: [httpd] ERROR: temboardagent.errors.HTTPError: Invalid session.
2022-02-22 14:37:30,682 temboardagent[14567]: [httpd] ERROR: {'error': 'Invalid session.'}
2022-02-22 14:37:30,682 temboardagent[14567]: [httpd] INFO: client: xxx.xxx.xxx.xxx request: "GET /profile?key=901ee76e7644a917b7ae1a5aaa7d7acb HTTP/1.1" 401 - 3.76ms
```

**Comment by bersace** [comment]
Hi @bsislow can you share me full DEBUG logs ? Did you upgrade o 7.10 ?

**Comment by bsislow** [comment]
DEBUG logs emailed.

We are planning on upgrading today actually.

After the upgrade we can let you know the outcome.

Let me know if the logs provide anything useful.

**Comment by bsislow** [comment]
This seems to be resolved after upgrading to 7.10. Thanks.
